packages/clay-multi-select/src/index.tsx

```diff
@@ -347,6 +334,10 @@ const ClayMultiSelect = React.forwardRef<HTMLDivElement, IProps>(
 							onFocus={(e) => {
 								onFocus(e);
 								setIsFocused(true);
+
+								if (sourceItems.length !== 0) {
+									setActive(true);
+								}
```

Comment:
Just out of curiosity and to understand this here. Since we are already doing the same in `useEffect` when it is focused, shouldn't this be more necessary here?